M2/Macaulay2/packages/GKMManifolds.m2

```diff
@@ -0,0 +1,1461 @@
+-*-------------------------------------------------------------------------------------------
+   Copyright 2020 Christopher Eur and Ritvik Ramkumar.
+
+   You may redistribute this file under the terms of the GNU General Public
+   License as published by the Free Software Foundation, either version 2 of
+   the License, or any later version.
+-------------------------------------------------------------------------------------------*-
+
+
+
+newPackage("GKMManifolds",
+	Version => "0.1",
+	Date => "August 1, 2020",
+	Authors => {
+	    {Name => "Chris Eur",
+       	     Email => "chriseur@stanford.edu",
+       	     HomePage => "https://web.stanford.edu/~chriseur"},
+	    {Name => "Ritvik Ramkumar",
+	     Email => "ritvik@math.berkeley.edu",
+	     HomePage => "https://math.berkeley.edu/~ritvik"}
+	    },
+	Headline => "a package for computations with GKM manifolds and moment graphs",
```

Comment:
Since I'm slowly working on adding keywords to packages, I might as well ask you now: what generic keyword do you think this package should be listed under? Would "Toric Geometry" be appropriate? If so, could you add the following option to `newPackage`:
```m2
	Keywords => {"Toric Geometry"},
```
Feel free to add more (e.g. "Matroids"), but for now only the first keyword is used for categorizing the packages.

Also, I think the convention is to not have "a package for" in the headline.